apps/demos/src/app/features/state/signal-state/signal-let.directive.ts

```diff
@@ -0,0 +1,54 @@
+import {
+  Directive,
+  effect,
+  EmbeddedViewRef,
+  Input,
+  OnDestroy,
+  OnInit,
+  Signal,
+  TemplateRef,
+  ViewContainerRef,
+} from '@angular/core';
+import { RxStrategyProvider } from '@rx-angular/cdk/render-strategies';
+import { Subject } from 'rxjs';
+
+@Directive({ selector: '[signalLet]', standalone: true })
```

Comment:
At this point, I'd just call it let directive as signals are going to be the recommended approach going forward

Reply:
I would reconsider the ideas in the template in general. The recent releases change a lot and we should adopt to the "modern way".